packages/squidex/schema/rules/rule4.json

```diff
@@ -1,13 +1,13 @@
 {
   "$schema": "./../__json/rule.json",
   "isEnabled": true,
-  "name": "Teams upated",
+  "name": "Teams Updated",
   "trigger": {
     "triggerType": "ContentChanged",
     "schemas": [
       {
         "schemaId": "teams",
-        "condition": "event.type == 'Created' || event.type == 'Updated'"
+        "condition": "(event.type == 'Updated' || event.type == 'Published') && event.status == 'Published'"
```

Comment:
see above comment - basically if we know for sure that the rule was here accidentally you are free to change it but otherwise i would check why it was here in the first place